ngraph/core/src/op/loop.cpp

```diff
@@ -297,14 +301,25 @@ void op::v5::Loop::validate_and_infer_types()
             }
         }
     }
+
+    NODE_VALIDATION_CHECK(this,
+                          get_output_size() == m_output_descriptions.size(),
+                          "Number of outputs must be the same as number of output descriptions");
 }
 
 std::shared_ptr<Node> op::v5::Loop::clone_with_new_inputs(const OutputVector& new_args) const
 {
     NGRAPH_OP_SCOPE(v5_Loop_clone_with_new_inputs);
+    // WA: first input description with index 0 or 1 means that Loop consructor will duplicate it in
+    // the inputs.
+    // When using visit_attributes() no duplication occurs, input_offset shall be decremented.
+    size_t input_offset = 2;
```

Comment:
the same comment as in validate_and_infer_types() func

Reply:
Already corrected.